ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2236,6 +2332,15 @@ - (BOOL)isEqual:(id)object {
             (_step == castObject.step));
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    //TODO: localization
+    NSDate *date = [NSDate dateWithTimeIntervalSince1970:[answer doubleValue]];
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    [dateFormatter setDateFormat:@"HH:mm"];
```

Comment:
Can you use `NSDateComponentsFormatter` instead, which gives you a nicer output.

http://nshipster.com/nsformatter/#nsdatecomponentsformatter